scripts/migrate-yaml.py

```diff
@@ -34,6 +34,9 @@ def parse_args():
                         help='The path to the file with defaults.')
     parser.add_argument('--output-file', dest='output_file', required=True,
                         help='The path to the new overrides file location')
+    parser.add_argument('--for-testing-only', dest='testing',
```

Comment:
given the people that use this option I'd rather warn off improper usage than have it be something like 'new-vars-only'